modules/candidate_list/php/candidatelistrowprovisioner.class.inc

```diff
@@ -0,0 +1,100 @@
+<?php declare(strict_types=1);
+/**
+ * This file implements a data provisioner to get all possible rows
+ * for the candidate list menu page.
+ *
+ * PHP Version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\candidate_list;
+
+/**
+ * This class implements a data provisioner to get all possible rows
+ * for the candidate list menu page.
+ *
+ * PHP Version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+clASs CandidateListRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
```

Comment:
`clASs` -> `class`

Reply:
oops.. fixed. (I had done a `s/as/AS/` to fix the capitalization of the SQL keyword and that substring slipped in..)